modules/core/src/main/scala/org/tessellation/infrastructure/snapshot/package.scala

```diff
@@ -0,0 +1,31 @@
+package org.tessellation.infrastructure
+
+import cats.data.NonEmptyList
+
+import org.tessellation.dag.snapshot.{GlobalSnapshot, SnapshotOrdinal}
+import org.tessellation.schema.height.{Height, SubHeight}
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.security.hash.Hash
+import org.tessellation.security.hex.Hex
+
+package object snapshot {
+
+  val genesisNextFacilitators: NonEmptyList[PeerId] = NonEmptyList
+    .of(
+      PeerId(Hex("peer1")),
+      PeerId(Hex("peer2")),
+      PeerId(Hex("peer3"))
+    )
+
+  val genesis: GlobalSnapshot =
+    GlobalSnapshot(
+      SnapshotOrdinal.MinValue,
+      Height.MinValue,
+      SubHeight.MinValue,
+      Hash("0"),
+      Set.empty,
+      Map.empty,
+      genesisNextFacilitators
+    )
+
+}
```

Comment:
Not sure if we should merge it to develop, given that it's a mock.

Reply:
I'm fine with merging mock/stub things into develop. It's out development branch and doesn't need to be production-ready. Yet, should be compilable. Hence, having such stubs helps to focus on a single thing and stubbing the others that can be implemented later.